cpp/data/sql_updates/ub_tools.18

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE metadata_presence_tracer MODIFY COLUMN metadata_field_name CHAR(3) NOT NULL;
```

Comment:
There has been another sql_update that has been merged to master in the meantime, please rename this one to `ub_tools.19`.

Reply:
Done